packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -123,4 +135,55 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
     (this[$sourceObject] as GLTFMaterial).emissiveFactor = rgb;
     this[$onUpdate]();
   }
+
+  setAlphaCutoff(value: number) {
+    (this[$sourceObject] as GLTFMaterial)
+    for (const material of this[$correlatedObjects] as
+         Set<MeshStandardMaterial>) {
+      // This little hack ignores alpha for opaque materials, in order to comply
+      // with the glTF spec.
+      if (!material.alphaTest && !material.transparent) {
+        material.alphaTest = -0.5;
+      } else if (this.alphaMode === 'MASK') {
+        material.alphaTest = value;
+      }
+    }
+    (this[$sourceObject] as GLTFMaterial).alphaCutoff = value;
+    this[$onUpdate]();
+  }
+
+  setAlphaMode(alphaMode: AlphaMode) {
+    (this[$sourceObject] as GLTFMaterial).alphaMode = alphaMode;
+    for (const material of this[$correlatedObjects] as
+         Set<MeshStandardMaterial>) {
+      if (alphaMode === 'BLEND') {
+        material.transparent = true;
+        material.depthWrite = false;
+        this[$onUpdate]();
+      }
+      if (alphaMode === 'OPAQUE') {
+        material.transparent = false;
+        material.depthWrite = true;  // default
+        this[$onUpdate]();
+      }
+      if (alphaMode === 'MASK') {
+        material.transparent = false;
+        material.depthWrite = true;  // default
+        this.setAlphaCutoff(
+            this.alphaCutoff !== undefined ? this.alphaCutoff : 0.5)
+      }
+    }
+  }
+
+  setDoubleSided(doubleSided: boolean) {
```

Comment:
The trouble here is that we do some special stuff for materials that are transparent and double-sided in order to get the triangle sorting working mostly right: https://github.com/google/model-viewer/blob/b5b86b3f681dc724fe9499c192041d2aa4e19fd8/packages/model-viewer/src/three-components/gltf-instance/ModelViewerGLTFInstance.ts#L77-L113

The trouble is that the special stuff is changes to the mesh objects that use this material, so I think you'll have to do a quick traversal to find meshes you may need to modify.